whois-update/src/main/java/net/ripe/db/whois/update/handler/validator/inet6num/Inet6numStatusValidator.java

```diff
@@ -35,57 +38,58 @@ public Inet6numStatusValidator(final Maintainers maintainers) {
     }
 
     @Override
-    public void validate(final PreparedUpdate update, final UpdateContext updateContext) {
+    public List<CustomValidationMessage> performValidation(final PreparedUpdate update, final UpdateContext updateContext) {
         switch (update.getAction()) {
```

Comment:
This is the same code that we have in InetnumStatusValidator. Maybe we should put that code in the interface that we are implementing as a Default...however, it is a bit weird to add as default something that is the same just in two subclass.

Reply:
Yes not correct way to put in default method , BusinessValidator is used for every type object not just inetnum and inetnum and inet6num validation should not be coupled together as these are separate object type with its own validation logic

Reply:
yes, agree...I suggest to use an util class to avoid this kind of duplications. However, I think that Ed prefers to go with some duplication than with utils class as well.